programs/editor/wodotexteditor.js

```diff
@@ -298,6 +301,7 @@ var Wodo = Wodo || (function () {
             //
             eventNotifier = new core.EventNotifier([
                 EVENT_UNKNOWNERROR,
+                EVENT_MODIFIEDCHANGED,
```

Comment:
Sadly the event "EVENT_METADATACHANGED" already has the suffix "CHANGED" without an underscore, so for consistency in 0.5.x I would like to use the same pattern. Could/should be changed for 0.6 for readability, agreed.

After some hours playing with that, IMHO the "is" prefix should stay syntactic sugar only for the getter of this boolean flag, and the event/signal should just get the flag name. Unless you have a strong opinion :)